client/guidestours/index.js

```diff
@@ -1,125 +1,20 @@
 /**
  * External dependencies
  */
-import React, { Component, PropTypes } from 'react'
+import React, { Component } from 'react'
```

Comment:
Sure :) Just want to keep client as lean as possible, we still have a fair bit of work to do there.